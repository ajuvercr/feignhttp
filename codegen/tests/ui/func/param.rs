use feignhttp_codegen::get;

// error: unsupported param parameter: `a: Vec < String >`
//    |  pub async fn send_get(#[param] a: Vec<String>) -> Result<String> {}
//    |                                 ^^^^^^^^^^^^^^

#[get("http://xxx")]
pub async fn send_get(#[param] a: Vec<String>) -> Result<String> {}

fn main() {}
