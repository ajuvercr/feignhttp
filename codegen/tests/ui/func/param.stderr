error: unsupported param parameter: `a: Vec < String >`
 --> tests/ui/func/param.rs:8:32
  |
8 | pub async fn send_get(#[param] a: Vec<String>) -> Result<String> {}
  |                                ^^^^^^^^^^^^^^
